### DISCUSSION TITLE
Stupid Question: How do I update an installed plugin?

### DISCUSSION THREAD

**Discussion by anoduck** [saulpw/visidata#1885] [Q&A] [answered]
It has been driving me crazy for over a year. I open visidata, it says I have 4 plugins with upgrades available, but I cannot find where it says how to perform an update in the docs. Obviously, `u` is not the correct answer.

**Comment by anjakefala** [comment, marked as answer]
It is cute that you tried `u`. =)

Thanks for finally coming here. This feature is likely to be removed in the next release as more and more plugins are brought in. Basically, it pops up if the version for a plugin does not match its latest version. 

You can use `open-plugins` to open the **PluginsSheet**, and then use `a` *y* to update an existing plugin. You can also manually replace them in `~/.visidata/plugins/`. 

You are right that until that message is removed, we should make sure the documentation is up to date!

**Reply by anoduck** [reply]
Well, `u` was logical at the time. Just executing add for updates worked for all the plugins compatible with my system. Still getting hangups with pillow for darkdraw, but that is nothing new. Thanks!
